rest-model/src/main/java/fi/otavanopisto/pyramus/rest/model/StudentContactLogEntry.java

```diff
@@ -8,13 +8,14 @@ public StudentContactLogEntry() {
     super();
   }
 
-  public StudentContactLogEntry(Long id, String text, String creatorName, OffsetDateTime entryDate, StudentContactLogEntryType type, Boolean archived) {
+  public StudentContactLogEntry(Long id, String text, String creatorName, OffsetDateTime entryDate, StudentContactLogEntryType type, Object comments, Boolean archived) {
```

Comment:
Shouldn't `Object comments` be more like `List<StudentContactLogEntryCommentRestModel> comments`? Would make the model easier to understand.